src/ElectricPokemon.java

```diff
@@ -0,0 +1,44 @@
+import javax.xml.namespace.QName;
+
+public class ElectricPokemon extends PokemonSuper {
+
+    private int electricity;
+    private int specialElectricity;
+
+
+    public ElectricPokemon(String name,int level, int hp, int xp, int electricity,int specialElectricity) {
+        super(name, level, hp, xp);
+        this.electricity = electricity;
+        this.specialElectricity = specialElectricity;
+    }
+
+    public void electricShock(){
+        System.out.println("You are attacked with an Electric Shock");
+    }
+
+    public void electroBall(){
+        System.out.println("You've been hit with an Electro Ball!");
+    }
+
+    @Override
+    public void attack() {
+        System.out.println("Attack is canceled.");
```

Comment:
Deze override geeft een compleet andere implementatie aan de methode. Dat is precies wat je van een override verwacht.